.github/workflows/rspec.yml

```diff
@@ -0,0 +1,30 @@
+name: rspec
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+jobs:
+  Rspec:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.6.9, 2.7, 3.0, 3.1, 3.2]
+    services:
+      redis:
+        image: redis:6.0
+        ports:
+          - 6379:6379
+    steps:
+      - name: Checkout action
+        uses: actions/checkout@v3
+      - name: Ruby setup
```

Comment:
@ruban-thilak please do it like this https://github.com/darkleaf/date_range_formatter/blob/master/.github/workflows/main.yml#L20-L23